dev/src/builtins.cpp

```diff
@@ -354,13 +354,17 @@ nfr("substring", "s,start,size", "SII", "S",
         return Value(ns);
     });
 
-nfr("string_to_int", "s", "S", "IB",
-    "converts a string to an int. returns 0 if no numeric data could be parsed."
-    "second return value is true if all characters of the string were parsed",
-    [](StackPtr &sp, VM &, Value &s) {
+nfr("string_to_int", "s", "SI?", "IB",
```

Comment:
note the `"s"` which are the names of the arguments, used in the automatic help generation. You added an argument, so this should probably be `"s,base"`. If you look at the comment above the assert that failed in CI, it says so :)

Reply:
Oops, I guess I was too quick. Will fix